mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -353,6 +353,43 @@ class ConstructorPattern final
   }
 };
 
+//===----------------------------------------------------------------------===//
+// MethodPattern - Converts SYCLMethodOpInterface to LLVM.
+//===----------------------------------------------------------------------===//
+
+static LogicalResult convertMethod(SYCLMethodOpInterface op,
+                                   ConversionPatternRewriter &rewriter) {
+  LLVM_DEBUG(llvm::dbgs() << "MethodPattern: Rewriting op: "; op.dump();
+             llvm::dbgs() << "\n");
+
+  SmallVector<mlir::Value> methodArgs(op->getOperands());
```

Comment:
SYCLRangeGetOp -> ValueRange ?

Reply:
I'm not sure that is suitable here, as we might need to modify the list of operands.

Reply:
ok